SCAutolib/models/gui.py

```diff
@@ -290,8 +290,10 @@ def __init__(self, wait_time: float = 5, res_dir_name: str = None):
         # otherwise the first character is not sent
         keyboard.send('enter')
 
-    def __enter__(self):
+        # create screen object to use from calls
         self.screen = Screen(self.screenshot_directory, self.html_file)
```

Comment:
Why this is moved outside of the enter function? I think we need a new fresh screen every time we initialize a test.

Reply:
I moved it to the __init__ so it would run outside of context manager use as well.  With this change, we should be able to use it either with or without context manager I believe.

Reply:
?

Reply:
We need to see that when we are over with the functionality and we try to fix the logging.